Enable request caching migration

Stanvirk opened the pull request.

Stanvirk committed:
Enable request caching migration

kevinchalet commented on guide/migration/20-to-30.md:
````diff
@@ -98,6 +98,20 @@ services.AddOpenIddict()
     });
 ```
 
+## Enable request caching
````
Should the section title be updated to something that doesn't imply it's a mandatory feature? Maybe "If applicable, use the new request caching activation methods"?

Stanvirk replied:
Good idea. Changed description. Is it better now?

kevinchalet commented:
Thanks for your PR!

Stanvirk committed:
Update 20-to-30.md

The feature migration description has now implicit mention that the feature is optional.

kevinchalet commented:
Merged, thanks for your contribution! :tada:
